src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
```

Comment:
Well done, this is a rather completed pull request. I like the indentation and comments. Clear and simple.